bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/BitcoindInstanceTest.scala

```diff
@@ -152,4 +153,49 @@ class BitcoindInstanceTest extends BitcoindRpcTest {
 
   }
 
+  it should "connect us to a remote bitcoind instance and ping it successfully" in {
+    val port = RpcUtil.randomPort
+    val rpcPort = RpcUtil.randomPort
+    val confStr = s"""
+                     |daemon=1
+                     |rpcauth=bitcoin-s:6d7580be1deb4ae52bc4249871845b09$$82b282e7c6493f6982a5a7af9fbb1b671bab702e2f31bbb1c016bb0ea1cc27ca
+                     |regtest=1
+                     |port=${RpcUtil.randomPort}
+                     |rpcport=${RpcUtil.randomPort}
+       """.stripMargin
+
+    val conf = BitcoindConfig(confStr, FileUtil.tmpDir())
+    val authCredentials =
+      BitcoindAuthCredentials.PasswordBased(username = "bitcoin-s",
+                                            password = "strong_password")
+    val instance =
+      BitcoindInstanceLocal(
+        network = RegTest,
+        uri = new URI(s"http://localhost:$port"),
+        rpcUri = new URI(s"http://localhost:$rpcPort"),
+        authCredentials = authCredentials,
+        datadir = conf.datadir,
+        binary = newestBitcoindBinary
+      )
+
+    val client = BitcoindRpcClient.withActorSystem(instance)
+    for {
+      _ <- startClient(client)
+      remoteInstance = BitcoindInstanceRemote(instance.network,
+                                               instance.uri,
+                                               instance.rpcUri,
+                                               instance.authCredentials,
+                                               instance.zmqConfig,
+                                               None)
```

Comment:
formatting is weird here and should use named args

Reply:
If you're in IntelliJ you can go to Settings > Editor > Code Style > Scala  and there is an option for `Reformat on file save`, this should prevent this, just make sure you set it to use `scalafmt` instead of the intellij defaults

![image](https://user-images.githubusercontent.com/15256660/130674231-1a2d7e25-0bb2-464d-ae11-23a94051e78a.png)

Reply:
I always keep getting scalafmt errors. Thanks for this!